quickstart/scripts/train-cloud.sh

```diff
@@ -27,6 +27,8 @@ export MODEL_NAME="quickstart" # change to your model name
 PACKAGE_PATH=./trainer # this can be a gcs location to a zipped and uploaded package
 export MODEL_DIR=gs://${BUCKET_NAME}/${MODEL_NAME}/model
 
+gsutil mb gs://${BUCKET_NAME}
```

Comment:
What if BUCKET already exists?

Reply:
It shows an error, but the script continues. But when it did not exist the training job failed (not sure how it worked before, I think I had the bucket already in my previous tests).